Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM java:8
 ENV CONF_HOME     /var/atlassian/confluence
 ENV CONF_INSTALL  /opt/atlassian/confluence
 ENV CONF_VERSION  5.9.3
+ENV CONF_PATH ""
```

Comment:
The entrypoint should check if the environment variable is set, or use a default value if the value is "empty", thus this is not needed.